### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by JoyBoy2398** [community/community#69481] [Repositories]
### Select Topic Area

Bug

### Body

Hello I can't stop that, even when I deleted it come back. Any idea?
![IMG20231007152002](https://github.com/community/community/assets/139411470/f5f941b7-5037-4c1d-9dfc-884be6ad13d0)

**Comment by AnkurBansal18** [comment]
@JoyBoy2398 , can you elaborate more of what problem are you facing exactly ?

**Comment by JoyBoy2398** [comment]
I can't delete it it come back everytime and apparently it have rob some work of other people. I have click on it and have install a dock modify for qubes who is my os and who just use my pc power. I was locking at it with this account and click on it Not on purpose and it does the same on is account so i think everyone who will connect to summer.fi who is a company from makerDAO will have the same problem, maybe delete their page??
